ouroboros-consensus/test-consensus/Test/Dynamic/Util.hs

```diff
@@ -250,10 +251,10 @@ leaderScheduleFromTrace (NumSlots numSlots) = LeaderSchedule .
         | nid `elem` xs = xs
         | otherwise     = nid : xs
 
-tooCrowded :: SecurityParam -> LeaderSchedule -> Bool
-tooCrowded k schedule = maxForkLength > maxRollbacks k
+tooCrowded :: SecurityParam -> NodeJoinPlan -> LeaderSchedule -> Bool
```

Comment:
What does it mean to be "too crowded"?

Reply:
I pushed a commit that updates this now-out-of-date terminology.